src/components/backupWallet/__tests__/BackupWallet.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { getWebRouterComponentWithMocks } from './__util__'
```

Comment:
i dont see why you need this, why not simply render <Backup screenProps={}>

Reply:
@sirpy I just copied tests folder implementation from Auth.js

Reply:
I checked if it is possible to just render it without getWebRouterComponentWithMocks wrapper. And we cannot do it as this wrapper providing required props for the rendered component. Without that props test just fails because the component throwing an error while rendering.